common/types.go

```diff
@@ -45,6 +45,9 @@ var (
 	hashT    = reflect.TypeOf(Hash{})
 	addressT = reflect.TypeOf(Address{})
 
+	// ZeroAddress represents the zero address value.
+	ZeroAddress = Address{}
+
```

Comment:
Meh. We already have `== (common.Address{})` all over, is a global zero address really so much better? IMO drop it
```suggestion
```

Reply:
done